File: PHP PowerPoint Examples/PlaceholdersApiDemo/getSlidesPlaceholders.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\PlaceholdersApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new PlaceholdersApi($configuration);

$name = "GetSlidesPlaceholders.pptx";
$password = null;
$folder = "Powerpoint";
$storage = null;
$slideIndex = 0;
$result = $apiInstance->getSlidesPlaceholders($name,$slideIndex, $password, $folder, $storage);
print_r($result);

File: Python PowerPoint Examples/PlaceholdersApiDemo/getSlidesPlaceholders.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.placeholders_api.PlaceholdersApi(configuration)

fileName = "getSlidesPlaceholders.pptx"
password = ""
storage = ""
slideIndex = 0
inputFolder = "input"
result = api.get_slides_placeholders(fileName, slideIndex,password=password, storage=storage,folder=inputFolder)
